docs/core/testing/unit-testing-with-nunit.md

```diff
@@ -0,0 +1,166 @@
+---
+title: Unit testing C# with NUnit and .NET Core
+description: Learn unit test concepts in C# and .NET Core through an interactive experience building a sample solution step-by-step using dotnet test and NUnit.
+keywords: NUnit, .NET, .NET Core
+author: rprouse
+ms.date: 12/01/2017
+ms.topic: article
+ms.prod: .net-core
+ms.devlang: dotnet
+---
+
+# Unit testing C# with NUnit and .NET Core
+
+This tutorial takes you through an interactive experience building a sample solution step-by-step to learn unit testing concepts. If you prefer to follow the tutorial using a pre-built solution, [view or download the sample code](https://github.com/dotnet/docs/blob/master/samples/core/getting-started/unit-testing-using-nunit/) before you begin. For download instructions, see [Samples and Tutorials](../../samples-and-tutorials/index.md#viewing-and-downloading-samples).
+
+### Creating the source project
+
+Open a shell window. Create a directory called *unit-testing-using-nunit* to hold the solution. Inside this new directory, run [`dotnet new sln`](../tools/dotnet-new.md) to create
+a new solution file for the class library and the test project. Next, create a *PrimeService* directory. The following outline shows the directory and file structure thus far:
+
+```
+/unit-testing-using-nunit
+    unit-testing-using-nunit.sln
+    /PrimeService
+```
+
+Make *PrimeService* the current directory and run [`dotnet new classlib`](../tools/dotnet-new.md) to create the source project. Rename *Class1.cs* to *PrimeService.cs*. To use test-driven development (TDD), you create a failing implementation of the `PrimeService` class:
+
+```csharp
+using System;
+
+namespace Prime.Services
+{
+    public class PrimeService
+    {
+        public bool IsPrime(int candidate)
+        {
+            throw new NotImplementedException("Please create a test first");
+        }
+    }
+}
+```
+
+Change the directory back to the *unit-testing-using-nunit* directory. Run [`dotnet sln add PrimeService/PrimeService.csproj`](../tools/dotnet-sln.md) to add the class library project to the solution.
+
+### Install the NUnit project template
+
+The NUnit test project templates need to be installed before creating a test project. This only needs to be done once. Run [`dotnet new -i NUnit3.DotNetNew.Template`](../tools/dotnet-new.md) to install the NUnit templates.
```

Comment:
Let's clarify that this is a per machine install. Meaning the command needs to be done once on each development machine where you'll create new NUnit projects.